scheduler/src/cook/mesos/util.clj

```diff
@@ -298,65 +298,59 @@
 ;; are set at the same time, in "real" time. This means that
 ;; jobs submitted after `start` will have been created after
 ;; expanded start
-(defn get-completed-jobs-by-user
-  "Returns all completed job entities for a particular user
-   in the specified timeframe, without a custom executor."
-  [db user start end limit state name-filter-fn]
-  (timers/time!
-    get-completed-jobs-by-user-duration
-    (let [;; Expand the time range so that clock skew between cook
-          ;; and datomic doesn't cause us to miss jobs
-          ;; 1 hour was picked because a skew larger than that would be
-          ;; suspicious
-          expanded-start (Date. (- (.getTime start)
+(defn get-jobs-by-user-and-state-and-submit
```

Comment:
Nitpick: the indentation below is off, it should be 2 spaces instead of 4.

Reply:
We should be consistent about style formatting.   Sham's nitpicks usually mean optional, but I don't think fixing this should be optional.